docs/content/configuration/storage.md

```diff
@@ -14,6 +14,7 @@ The Athens proxy supports many storage types:
 1. [AWS S3](#aws-s3)
 1. [Minio](#minio)
```

Comment:
Both Alibaba OSS and Digital Ocean use minio. Is it possible to add a note saying any minio compatible storage can be included. Can you also add a note that if a user is configuring a minio compatible storage, then a bug report should be filed?

This would cover up all the use cases.